jaggery-modules/rxt/module/scripts/asset/asset.js

```diff
@@ -261,44 +261,46 @@ var asset = {};
      * @lends AssetManager.prototype
      */
     AssetManager.prototype.create = function(options) {
-        metrics.start(this.constructor.name,'create');
-        validateRequiredFeilds(this.type, options , this.rxtManager);
-        var isDefault = false;
-        if ((options.hasOwnProperty(constants.Q_PROP_DEFAULT)) && (options[constants.Q_PROP_DEFAULT] === true)) {
-            delete options[constants.Q_PROP_DEFAULT];
-            isDefault = true;
-        }
-        if(options.attributes.hasOwnProperty(constants.ASSET_PROVIDER)){
-            options.attributes[constants.ASSET_PROVIDER] = options.attributes[constants.ASSET_PROVIDER].replace('@', ':');
-        }
-        
-        var namespaceAttribute = this.rxtManager.getNamespaceAttribute(this.type);
+        try {
+            metrics.start(this.constructor.name,'create');
+            validateRequiredFeilds(this.type, options , this.rxtManager);
+            var isDefault = false;
+            if ((options.hasOwnProperty(constants.Q_PROP_DEFAULT)) && (options[constants.Q_PROP_DEFAULT] === true)) {
+                delete options[constants.Q_PROP_DEFAULT];
+                isDefault = true;
+            }
+            if(options.attributes.hasOwnProperty(constants.ASSET_PROVIDER)){
```

Comment:
Thanks @splinter , I have fixed it in [this commit](https://github.com/tmkasun/carbon-store/commit/a5917ea67af4d60979a9f5f59636c7672843645b)